src/NosCore.GameObject/Providers/InventoryService/InventoryService.cs

```diff
@@ -474,6 +474,41 @@ public InventoryItemInstance RemoveItemAmountFromInventory(short amount, Guid id
             return null;
         }
 
+        public List<InventoryItemInstance> RemoveItemAmountFromInventoryByVNum(short amount, short itemVNum)
+        {
+            List<InventoryItemInstance> result = new List<InventoryItemInstance>();
+            var stayAmount = amount;
+            foreach (var itemInstance in this.Select(s => s.Value)
+                .Where(w => w.ItemInstance.ItemVNum == itemVNum)
+                .OrderBy(o => o.ItemInstance.Amount))
+            {
+                if (stayAmount == 0)
+                {
+                    break;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Refactor the code in order to remove this break statement.](https://app.codacy.com/app/NosCoreIO/NosCore/pullRequest?prid=4518428)